src/objecttypes/api/v1/openapi.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
-  title: Objects API
-  version: 1.1.0
+  title: Objecttypes API
+  version: v1
```

Comment:
Hmm, where did the exact version go?

I see I missed this for the Objects API as well: https://github.com/maykinmedia/objects-api/blob/master/src/objects/api/v2/openapi.yaml#L4

Reply:
I've excluded full versions because the default version generation showed only 1 full version. Now I fixed it with a postprocessing hook. The version will look like `1.1.1 (v1)` or `2.0.0 (v2)`